jest.config.js

```diff
@@ -25,19 +25,15 @@ module.exports = {
             branches: 0
```

Comment:
When we update these values?

Reply:
Do all tests already 100% cover renovation code?

Reply:
I think no. But we can increase these properties to some values to get rid of adding widgets without testing. And we can ask some guys to increase code coverage rating.

Reply:
@ksercs will do it